pkg/editor/windows.go

```diff
@@ -0,0 +1,9 @@
+// +build windows
+
+package editor
+
+const (
+	defaultShell     = "cmd"
+	shellCommandFlag = "/C"
+	defaultEditor    = "notepad"
```

Comment:
Done research. There is no other preinstalled editor that does support json formatting etc.